checka11y.css

```diff
@@ -87,3 +87,138 @@ a[target = "_blank"]::after /* Links that open a new page/tab target="_blank" */
   content: " WARNING: Links that open in a new tab or window should be communicated to the user, ensure the user knows of this behavior" !important;
   font-family: var(--checka11y-font, cursive);
 }
+
```

Comment:
While this work is to do with links, due to the size of this feature, I think we can create a new section here:

```css
/**
 * Invalid HTML nested inside <a>
 */
```